Assets/MRTK/Core/MRTK.Core.asmdef

```diff
@@ -2,7 +2,8 @@
     "name": "Microsoft.MixedReality.Toolkit",
     "references": [
         "Microsoft.MixedReality.Toolkit.Async",
-        "Microsoft.MixedReality.Toolkit.Editor.Utilities"
+        "Microsoft.MixedReality.Toolkit.Editor.Utilities",
+        "Unity.XR.Management"
```

Comment:
Not a huge fan of adding a dependency like this to the core, but I'm not sure another way around it with the APIs Unity provides...

Reply:
same here. we DO have similar entries in some of our other packages

Reply:
Some make sense to have the references (like providers that are built specifically on XR SDK), but the core is a bit different imo

Reply:
We will be saying goodbye to this soon in 3.0 and also it feels more like a Unity-created issue so I'd vote for including that dependency. After all XR is pretty core to what we do, it's just not as core to Unity.